documentation/onboard-form-template.md

```diff
@@ -0,0 +1,47 @@
+# Teacher services infra - Service onboarding form
+- Service name?
+   - e.g. Calculate teacher pay
+- Domains  ? e.g.
+    - school-placements-production.teacherservices.cloud
+    - getintoteaching.education.gov.uk and
+    - get-information-schools.service.gov.uk
+- Service short name? e.g. ctp
+- Repository? e.g. https://github.com/DFE-Digital/itt-mentor-services
+- Environments? e.g. develoment, staging, production
+- Review apps? Yes or No
+- Cost centre(using format cost centre/act code )? e.g. 10167/101304
+- Activity code? e.g
+    - placements: 101423
+    - track: 101056
+- List of developers who will need access to Azure? Do they use a DfE laptop or BYOD? e.g
+    - James Beagle - BYOD
+    - Clinton Morris - DfE laptop
+- GDS assessed? Yes or No
+
+- Slack channels? e.g
+    - Product: #itt-mentor-services
+    - Tech: #itt-mentor-services-devs
+- Timeline? e.g
```

Comment:
This could be more concise and clearer
- Timeline? When is each environment needed? What are the planned [project phases](https://www.gov.uk/service-manual/agile-delivery)?

And remove the examples